src/main/java/gitflow/actions/RepoActions.java

```diff
@@ -71,10 +71,9 @@
         }
 
         //gitflow not setup
-        if (branchUtil.hasGitflow()!=true){
+        if (!branchUtil.hasGitflow()){
```

Comment:
Please retain the existing conditional formatting. It's more verbose for readability purposes